docs/course_materials/compound_data_types.html

```diff
@@ -275,10 +228,12 @@ <h3><b>Basics</b></h3>
                 </section>
                 <section>
                     <h3><b>Example</b></h3>
-                    <pre><code data-trim data-noescape class="hljs">
-sports = {'Ralph' : 'Football',
-          'Michael' : 'Basketball',
-          'Shirly' : 'Baseball'}
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+sports = {
```

Comment:
I think we should stay coherent, as we create a list as lst =  [...] (on the same line).
I think the way before is a better one.